src/structure_new/src/model_evaluator/4C_structure_new_model_evaluator_multiscale.hpp

```diff
@@ -117,8 +126,11 @@ namespace Solid
       void post_output() override{};
 
      private:
-    };
+      std::shared_ptr<Core::IO::DiscretizationVisualizationWriterMesh> vtu_writer_ptr_;
 
+      //! visualization parameters
+      Core::IO::VisualizationParameters macro_visualization_params_;
+    };
```

Comment:
Why is the macro visualisation not handled by the default structure model evaluator?